.github/workflows/maven.yaml

```diff
@@ -1,7 +1,7 @@
 # This workflow will build a Java project with Maven
 # For more information see: https://help.github.com/actions/language-and-framework-guides/building-and-testing-java-with-maven
 
-name: mvn verify
+name: ./mvnw verify
```

Comment:
You can run `mvn wrapper:wrapper -Dmaven=3.9.6` prior to running the Maven wrapper to ensure the project is set up with the right version of Maven to build the project, and you can enforce the version with the enforcer plugin by setting: `<revelc.min-build-mvn>3.9.0</revelc.min-build-mvn>` (there's already a rule in the parent POM for that, so we just need to override the version property.

Reply:
Actually, I see that the eclipse-plugin custom lifecycle defined by the tycho-maven-plugin doesn't include a validate phase for the maven-enforcer-plugin to bind to, which it does in the parent POM. We can still use it to enforce, but we'd have to add it to this POM and bind it to a different phase that actually exists in this lifecycle. But, I don't think that part is strictly necessary, since the enforcement of Maven 3.9.0 is already done by tycho itself and the error message is already pretty clear.